remill/Arch/AArch64/Arch.cpp

```diff
@@ -487,13 +487,19 @@ static void AddExtendRegOperand(Instruction &inst, RegClass rclass,
 
 static void AddImmOperand(Instruction &inst, uint64_t val,
                           ImmType signedness=kUnsigned,
```

Comment:
I suggest changing thing from `signedness` to `imm_type`, and then adding in a `kFloat` to the `ImmType`